k9mail/src/main/java/com/fsck/k9/helper/MailToHelper.java

```diff
@@ -0,0 +1,157 @@
+package com.fsck.k9.helper;
+
+import android.net.MailTo;
+import android.net.Uri;
+import android.util.Log;
+
+import com.fsck.k9.K9;
+import com.fsck.k9.exceptions.ParseException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * MailTo helper class
+ */
+public class MailToHelper {
```

Comment:
Try to avoid the "Helper" suffix. Since we're not using `android.net.MailTo` calling this class `MailTo` is fine.